app/helpers/template_helpers.py

```diff
@@ -216,7 +216,7 @@ def get_survey_config(
     )
 
 
-def render_template(template: str, **kwargs: Union[str, Mapping]) -> str:
+def render_template(template: str, **kwargs: Union[Optional[str], Mapping]) -> str:
```

Comment:
As per style guide, for more than 2 types it should be `Union[None, str, Mapping]`

Reply:
Change added to fit style guide 👍